### PR TITLE
Fix Incorrect scroll edge appearance

### PR DESCRIPTION
**Pull request by Moux**
Address iOS 15 changes to standard and scroll edge appearances for navigation bars.

### CODE REVIEW

File: Sources/UI/ScreenViewController.swift [hunk 1]
```diff
@@ -168,6 +168,8 @@ open class ScreenViewController: UICollectionViewController, UICollectionViewDat
             return UIColor(red: (247 / 255), green: (247 / 255), blue: (247 / 255), alpha: 1)
         }()
         
+        navigationBar.barTintColor = backgroundColor
```
Comment (orospakr) [inline Sources/UI/ScreenViewController.swift:171]: does this change behaviour on iOS <15?

**Review by orospakr — CHANGES_REQUESTED** [review]

File: Sources/UI/ScreenViewController.swift (hunk 1 above)
Comment (orospakr) [reply Sources/UI/ScreenViewController.swift:171]: ah I get it now

**Comment by Moux** [comment]
Results with 2020 Pro Bowl Vote:

Experience not scrolled:
Screen Shot 2021-12-21 at 3 02 58 PM

Experience scrolled:

Screen Shot 2021-12-21 at 3 03 08 PM

**Comment by Moux** [comment]
Results with Bill - Holiday Gift Guide Final:

Experience not scrolled:

Screen Shot 2021-12-21 at 3 05 41 PM

Experience scrolled:

Screen Shot 2021-12-21 at 3 05 54 PM

**Review by orospakr — APPROVED** [review]

**Pull request merged**